### ISSUE TITLE
Add GitHub Desktop Contribution Instructions

### ISSUE THREAD

**Issue by PradnyaGaitonde** [Bhanu-code/db-drawer#48]
I am interested in contributing to the project by adding detailed instructions on how to contribute using GitHub Desktop.

I believe that by providing clear and comprehensive guidance, we can encourage more contributors to participate in the project, especially those who prefer using GitHub Desktop for their contributions.

Could you please assign me the issue related to adding GitHub Desktop contribution instructions under gssoc24? I am excited about the opportunity to contribute to the project and help streamline the contribution process for others.

**Comment by Bhanu-code** [comment]
@Bhanu-code  would you mind joining a meeting and explaining me about the feature before raising PR?

**Comment by PradnyaGaitonde** [comment]
@Bhanu-code Yeah sure, I want to add detailed instructions for contributors to pull requests using the GitHub Desktop application so that they do not face any difficulty.

**Comment by Bhanu-code** [comment]
@PradnyaGaitonde ping me in GSSoC general channel
